### ISSUE TITLE
Ogden Clinic Timeout Error

### ISSUE THREAD

**Issue by Batman313v** [fastenhealth/fasten-onprem#274]
I am receiving the following error while trying to sync the Ogden clinic
```json
{
  "source_id": "4834742c-f381-4f57-9e33-8adb9cc2675f",
  "source_type": "ogden-clinic",
  "error_data": {
    "final": "an error occurred while bulk importing resources from source: Get \"https://fhir4.eclinicalworks.com/fhir/r4/JJJGBA/Patient/XXXXXXXXXXXXXXXXXXXXXXXXXXXXXX\": context deadline exceeded (Client.Timeout exceeded while awaiting headers)"
  }
}
```

**Comment by AnalogJ** [comment]
Can you confirm that the website for your provider is https://www.ogdenclinic.com/ and your patient portal looks like https://mycw62.ecwcloud.com/portal7937/jsp/100mp/login_otp.jsp ?

**Comment by Batman313v** [comment]
Yep, those are correct

**Comment by AnalogJ** [comment]
Hey @Batman313v this might require a debugging session. I was able to verify that my sandbox connection to  eClinicalWorks is having issues as well. 

Any chance you're on [Discord](https://discord.gg/Bykz6BAN8p)? I can update you once I figure out whats going on.

**Comment by Batman313v** [comment]
> Hey @Batman313v this might require a debugging session. I was able to verify that my sandbox connection to  eClinicalWorks is having issues as well. 
> 
> Any chance you're on [Discord](https://discord.gg/Bykz6BAN8p)? I can update you once I figure out whats going on. 
>  

Yep, I'll send you a message

**Comment by dpiessens** [comment]
Just stumped across the project, very cool! Might need to ask your clinic which version of eCW they are on. If it's still version 11 FHIR isn't available.

**Comment by AnalogJ** [comment]
@dpiessens do you know if the eCW version is specified in the Capability Statement/Conformance document anywhere?

https://fhir4.eclinicalworks.com/fhir/r4/JJJGBA/metadata

I took a look but didn't see any 2 digit versions, only

```
   <id value="eCWFhirFacade"/>
...
   <version value="2.0"/>
   <name value="eCW FHIR Facade"/>
   ...
   <publisher value="eClinicalWorks"/>
   ...
   <software>
      <name value="eCW FHIR Facade"/>
      <version value="1.6"/>
   </software>
```

**Comment by dpiessens** [comment]
No, just that v11 doesn't have FHIR at all. My org uses it at the clinic level, and ran into it a few times.

**Comment by AnalogJ** [comment]
I think the solution here is to allow users to customize the Timeout as discussed in #357 , with a fallback to requesting resource types individually, without a bulk export.
